src/bnb_trading/fibonacci.py

```
@@ -165,76 +166,80 @@
+            swing_low_price = lookback_data.loc[swing_low_idx, "Low"]
             swing_low_pos = lookback_data.index.get_loc(swing_low_idx)
-            
+
             # Проверяваме дали swing е достатъчно голям
             swing_size = abs(swing_high_price - swing_low_price) / swing_low_price
-            
+
             if swing_size < self.min_swing_size:
-                logger.warning(f"Swing размер {swing_size:.1%} е под минимума {self.min_swing_size:.1%}")
+                logger.warning(
+                    f"Swing размер {swing_size:.1%} е под минимума {self.min_swing_size:.1%}"
```

Ruff (0.12.2)

String contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF001)